test/benchmarks/material/radio/radio.perf-spec.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {$, browser} from 'protractor';
+import {runBenchmark} from '@angular/dev-infra-private/benchmark/driver-utilities';
+
+describe('radio button performance benchmarks', () => {
+  beforeAll(() => {
+    browser.rootEl = '#root';
+  });
+
+  it('renders two radio buttons', async() => {
+    await runBenchmark({
+      id: 'render-two-radio-buttons',
+      url: '',
+      ignoreBrowserSynchronization: true,
+      params: [],
+      prepare: async () => await $('#hide-two').click(),
+      work: async () => await $('#show-two').click(),
+    });
+  });
+
+  it('renders ten radio buttons', async() => {
+    await runBenchmark({
+      id: 'render-ten-radio-buttons',
+      url: '',
+      ignoreBrowserSynchronization: true,
+      params: [],
+      prepare: async () => await $('#hide-ten').click(),
+      work: async () => await $('#show-ten').click(),
+    });
+  });
+
+  it('clicks between radio buttons', async() => {
```

Comment:
Is the test here to measure how clicking performs between two buttons (inclusive the first click), or do you just want to measure the click when a radio is focused already? Can you explain why these might be useful measurements?

Reply:
This test measures how long the animation for clicking a radio button is. It measures a single click so the test keeps toggling between the two options. If this isn't a useful metric then it can be deleted

Reply:
Got it. Maybe we can clarify this better by naming the test `changing between radio buttons`?